pkg/issuer/acme/dns/util/wait_test.go

```diff
@@ -177,3 +177,24 @@ func TestResolveConfServers(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateCAA(t *testing.T) {
+	// google installs a CAA record at google.com
+	// ask for the www.google.com record to test that
+	// we recurse up the labels
+	err := ValidateCAA("www.google.com", []string{"letsencrypt", "pki.goog"}, false)
```

Comment:
If the directory responds with 2 different CAA names, does that mean that the CAA record can be set to either one of the names?

Reply:
Correct